src/typing/type_inference_js.ml

```diff
@@ -982,7 +977,7 @@ and statement_decl cx = Ast.Statement.(
       let r = mk_reason (spf "module %s" name) loc in
       let t = Flow_js.mk_tvar cx r in
       Hashtbl.replace cx.type_table loc t;
-      Env_js.init_env cx (spf "$module__%s" name) (create_env_entry t t (Some loc))
+      Env_js.init_env cx (spf "$module__%s" name) (create_env_entry t t (Some loc)) VarScope (* TODO: hoist behavior *)
```

Comment:
@jeffmo Modules is an interesting case. Does it even make sense to argue non-global modules? Does this argue for a GlobalScope alternative to the scope_kind type?